### ISSUE TITLE
Replace FLOAT types with NUMERIC

### ISSUE THREAD

**Issue by miike** [snowplow-incubator/snowplow-bigquery-loader#8]
By replacing some of the current transaction FLOAT types with NUMERIC we avoid any floating point arithmetic issues that we can run into if using BigQuerys FLOAT64.

**Comment by chuwy** [comment]
https://github.com/snowplow-incubator/snowplow-bigquery-loader/commit/8653b16a0b33f52e2a9f8ab0a96574f1abf7ffc4

**Comment by dilyand** [comment]
@chuwy Can this be closed?

**Comment by chuwy** [comment]
No, I think we still need this change. BQ Loader is shipped with schema-ddl 0.9.0 since 0.2.0, which means it creates new columns with `NUMERIC` type https://github.com/snowplow/iglu/issues/426. Now we need to cherry-pick https://github.com/snowplow-incubator/snowplow-bigquery-loader/commit/8653b16a0b33f52e2a9f8ab0a96574f1abf7ffc4 to reflect this standard atomic events fields.
